packageheadless.json

```diff
@@ -1,68 +1,27 @@
 {
   "name": "shield",
-  "version": "0.0.8",
+  "version": "0.0.14",
   "description": "Cyber security shield",
   "productName": "UA Cyber SHIELD",
   "author": "Mykola Zhyhallo <mzhyhall@gmail.com>",
   "private": true,
   "repository": "github:opengs/uashield",
   "scripts": {
     "build:headless": "esbuild ./headless.ts --platform=node  --bundle --minify --sourcemap --sources-content=false --keep-names --target=node16 --outfile=./build/headless/headless.js",
-    "start:headless": "NODE_ENV=production node --enable-source-maps ./build/headless/headless.js",
-    "lint": "eslint --ext .js,.ts,.vue ./",
-    "test": "echo \"No test specified\" && exit 0"
+    "start:headless": "NODE_ENV=production node --enable-source-maps ./build/headless/headless.js"
   },
   "dependencies": {
-    "@types/node-fetch": "^2.6.1",
-    "@types/superagent": "^4.1.15",
-    "axios": "^0.21.1",
+    "axios": "^0.26.1",
     "axios-https-proxy-fix": "^0.17.1",
-    "socks-proxy-agent": "^6.2.0-beta.0",
-    "core-js": "^3.6.5",
-    "fs": "^0.0.1-security",
-    "path": "^0.12.7",
-    "simple-ddos": "^2.0.4",
-    "superagent": "^7.1.1",
-    "vue": "^3.0.0",
-    "vue-i18n": "^9.0.0",
-    "vue-router": "^4.0.0",
-    "user-agents": "^1.0.943",
-    "vuex": "^4.0.1",
-    "uuid4": "^2.0.2",
-    "universal-analytics": "^0.5.3"
```

Comment:
headless version probably will not build. at least several packages are required. try delete node_modules and install from packageheadless.json

Reply:
you are right, missed it, messed up while switching between these files